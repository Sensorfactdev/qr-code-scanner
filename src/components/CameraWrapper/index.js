import styled from 'styled-components';

export default styled.div`
  position: relative;

  &:after {
    content: "";
    display: ${({ showAimAssist }) => showAimAssist ? 'block' : 'none'};
    position: absolute;
    top: 30%; left: 25%;
    width: 50%; height: 25%;
    border: 5px solid limegreen;
  }
`;
