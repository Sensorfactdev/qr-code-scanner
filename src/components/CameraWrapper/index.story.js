import React from 'react';
import { storiesOf } from '@storybook/react';
import CameraWrapper from './index';

storiesOf('CameraWrapper', module)
  .add('default view', () => (
    <CameraWrapper />
  ));
