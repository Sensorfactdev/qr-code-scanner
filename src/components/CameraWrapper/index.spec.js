import React from 'react';
import renderer from 'react-test-renderer';
import CameraWrapper from './index';

describe('CameraWrapper', () => {
  it('should render', () => {
    const component = renderer.create(
      <CameraWrapper />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
