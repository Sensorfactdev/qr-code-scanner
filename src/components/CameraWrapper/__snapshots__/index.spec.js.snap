// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CameraWrapper should render 1`] = `
<div
  className="sc-bdVaJa hgRUri"
/>
`;
